docs/www/guide-wasm-elastic.md

```diff
@@ -0,0 +1,189 @@
+# Redpanda wasm function + Elastic Search
+One of the richest wasm function case is indexing information to [ElasticSeach](https://www.elastic.co/elasticsearch) from Redpanda's records. for this we need to requirements:
+
+1. Redpanda > v21.1.5
+2. ElasticSearch service, in this case we use the [docker image](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html), in localhost:9200
+
+## Enable wasm on Redpanda
+Add a `coproc_enable: true` in redpanda.yaml
+
+```yaml
+   # Skips most of the checks performed at startup (i.e. memory, xfs)
+   # not recomended for production use
+   developer_mode: true
+   enable_coproc: true # enable wasm
+```
+
+## Create Wasm function
+for creating a wasm function we can use `rpk` tools.
```

Comment:
```suggestion
To create a wasm function we can use `rpk` tools. It will generate a template giving us a simple build system and startup example.
```

Reply:
Please show the specific `rpk wasm` command and its output.